test/common/fam_test_config.h.in

```diff
@@ -38,9 +38,12 @@
 #include <string.h>
 #include <fam/fam.h>
 
+#define RAID1 NULL
+#define RAID5 NULL
+#define NONE NULL
```

Comment:
@faizan-barmawer What do we do in test cases. Should we change all test cases to use NULL in place of NONE ?